cloudinit/helpers.py

```diff
@@ -317,7 +318,9 @@ def items(self):
         return list(self.registered.items())
 
 
-class Paths(object):
+class Paths(persistence.CloudInitPickleMixin, object):
```

Comment:
No need to inherit from `object` at all in Python 3:

```suggestion
class Paths(persistence.CloudInitPickleMixin):
```